### PR TITLE
Clarifications about running dbt init --profile

### PR DESCRIPTION
**Pull request by victorrgez**
Clarifications about running dbt init --profile

## What are you changing in this pull request and why?

-Clarify the use of `dbt init` when using an existing `profiles.yml` file. The current docs are not clear enough and one may think you have to provide the path to `profiles.yml` as an argument, for example, `dbt init --profile profiles.yml` when, in fact, you need to pass the actual name of your profile.

<br>

**REMOVED CHANGES:**

-~~Clarify that you cannot pass the `--profile` flag  if you already have a dbt_project.yml file as the error returned (`Can not init existing project with specified profile, edit dbt_project.yml instead`) may be confusing~~

-~~Change VersionBlock version from 1.7 to 1.8~~

## Checklist

- [X] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.

- [X] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).

- [X] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

- [X] Need review of the doc changes as this is my first contribution

### CODE REVIEW

File: website/docs/reference/commands/init.md [hunk 1]
```diff
@@ -17,13 +17,13 @@ Then, it will:
 - Create a new folder with your project name and sample files, enough to get you started with dbt
 - Create a connection profile on your local machine. The default location is `~/.dbt/profiles.yml`. Read more in [configuring your profile](/docs/core/connect-data-platform/connection-profiles).
 
-<VersionBlock firstVersion="1.7">
```
Comment (aranke) [inline website/docs/reference/commands/init.md:20]: I don't think this is correct, it's in the changelog for 1.7.0

https://github.com/dbt-labs/dbt-core/blob/1.7.latest/CHANGELOG.md#fixes-2

File: website/docs/reference/commands/init.md [hunk 2]
```diff
@@ -17,13 +17,13 @@ Then, it will:
...
+<VersionBlock firstVersion="1.8">
 
-When using `dbt init` to initialize your project, include the `--profile` flag to specify an existing `profiles.yml` as the `profile:` key to use instead of creating a new one. For example, `dbt init --profile`.
+When using `dbt init` to initialize your project, include the `--profile` flag to specify an existing `profiles.yml` as the `profile:` key to use instead of creating a new one. For example, `dbt init --profile profile_name`.
 
 
 
-If the profile does not exist in `profiles.yml` or the command is run inside an existing project, the command raises an error.
+If the profile does not exist in `profiles.yml` or the command is run inside an existing project (that is, if `dbt_project.yml` already exists), the command raises an error.
```
Comment (aranke) [inline website/docs/reference/commands/init.md]: This doesn't sound right, `--profile` overrides the value in `dbt_project.yml`

https://github.com/dbt-labs/dbt-core/blob/main/core/dbt/cli/params.py#L336-L340

File: website/docs/reference/commands/init.md [hunk 3]
```diff
@@ -17,13 +17,13 @@ Then, it will:
 - Create a new folder with your project name and sample files, enough to get you started with dbt
 - Create a connection profile on your local machine. The default location is `~/.dbt/profiles.yml`. Read more in [configuring your profile](/docs/core/connect-data-platform/connection-profiles).
 
-<VersionBlock firstVersion="1.7">
+<VersionBlock firstVersion="1.8">
 
-When using `dbt init` to initialize your project, include the `--profile` flag to specify an existing `profiles.yml` as the `profile:` key to use instead of creating a new one. For example, `dbt init --profile`.
+When using `dbt init` to initialize your project, include the `--profile` flag to specify an existing `profiles.yml` as the `profile:` key to use instead of creating a new one. For example, `dbt init --profile profile_name`.
```
Comment (aranke) [inline website/docs/reference/commands/init.md:22]: Great catch!

**Review by aranke — COMMENTED** [review]
1 change looks good, not sure about other 2

**Comment by victorrgez** [comment]
Sorry about the other two changes. They were the result of a misunderstanding. 

Thank you @aranke for your time reviewing the PR!

**Review by aranke — APPROVED** [review]
LGTM, ty!

**Comment by victorrgez** [comment]
Hi @aranke 

I'm not authorised to merge the PR, could you please do it or is there something else left to do first?

thanks!

**Comment by mirnawong1** [comment]
hey @victorrgez and @aranke -- thank you so much for contributing and merging this PR for you now, Victor!

**Pull request merged**